doc/classes/Resource.xml

```diff
@@ -23,7 +23,7 @@
 			<argument index="0" name="subresources" type="bool" default="false">
 			</argument>
 			<description>
-				Duplicates the resource, returning a new resource. By default, sub-resources are shared between resource copies for efficiency, this can be changed by passing [code]true[/code] to the [code]subresources[/code] argument.
+				Duplicates the resource, returning a new resource. By default, sub-resources are shared between resource copies for efficiency, this can be changed by passing [code]true[/code] to the [code]subresources[/code] argument which will copy the subresources.  In the case of [code]subresources[/code] being [code]true[/code] duplicate does not perform a deep copy - nested resources within subresources will not be duplicated and will still be shared.
```

Comment:
```suggestion
				Duplicates the resource, returning a new resource. By default, sub-resources are shared between resource copies for efficiency, this can be changed by passing [code]true[/code] to the [code]subresources[/code] argument which will copy the subresources. If [code]subresources[/code] is [code]true[/code], this method will only perform a shallow copy. Nested resources within subresources will not be duplicated and will still be shared.
```